src/cc_catalog_airflow/dags/provider_api_scripts/common/storage/image.py

```diff
@@ -307,19 +333,35 @@ def _flush_buffer(self):
         if buffer_length > 0:
             logger.info(
                 'Writing {} lines from buffer to disk.'
-                .format(buffer_length)
+                  .format(buffer_length)
```

Comment:
This is the only formatting change I disagree with.  Please revert it.

Reply:
I have reverted it but pycodestyle complains about these lines saying continuation line unaligned for hanging indent.